numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -78,8 +78,8 @@ def foo(x, y):
             i = cuda.grid(1)
             x[i] += bar(y[i])
 
-        A = np.array([123])
-        B = np.array([321])
+        A = np.array([123]).astype(np.int32)
+        B = np.array([321]).astype(np.int32)
```

Comment:
perhaps use `np.array(<arg>, dtype=np.int32)` so as to not create a temporary?

Reply:
Done.